web_widget_uom/models/uom_uom.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```

Comment:
is this the right copyright?

Reply:
Started from this pr: https://github.com/OCA/web/pull/1279. So i thought better not change copyright